### ISSUE TITLE
TypeError: Cannot set property 'open' of undefined

### ISSUE THREAD

**Issue by atrauzzi** [systemjs/plugin-babel#80]
Getting this error when trying to build my JSPM based app:

```
$ jspm build src/App/FormManager/main.tsx Content/build/formmanager.js
     Creating the single-file build for src/App/FormManager/main.tsx...

err  (SystemJS) Cannot set property 'open' of undefined
  TypeError: Cannot set property 'open' of undefined
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:30442:24
      at a (C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:2434)
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:1872
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:30611:16
      at a (C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:2434)
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:1872
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:30709:25
      at a (C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:2434)
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:1872
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:33551:45
      at a (C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:2434)
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:1872
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:35294:15
      at a (C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:2434)
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:1:1872
      at C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/systemjs-babel-node.js:42755:24
  Evaluating C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/plugin-babel.js
  Error loading C:/project/Content/jspm_packages/npm/systemjs-plugin-babel@0.0.23/plugin-babel.js
PS C:\project>
```

Not at all sure what to do to resolve this as all the errors happen in framework/internal code.

**Comment by atrauzzi** [comment]
Seems to be when running on Windows, some terminal stuff is breaking in `systemjs-babel-node.js`.

**Comment by guybedford** [comment]
This should be fixed now in 0.0.24.
